maintainers/working_groups/learning_journey/tutorial-outlines/first-steps-trivial-builders.md

```diff
@@ -0,0 +1,37 @@
+# First Steps - Trivial Builders
+
+## Assumptions
+
+* The reader knows what a shell environment is.
```

Comment:
clarify what you mean by shell environment - more specific?
Does it mean that they have seen shell.nix or more some familiarity with nix-shell?